### ISSUE TITLE
Provide more explaination about a service resources 

### ISSUE THREAD

**Issue by qgau** [InseeFrLab/onyxia#675]
Hello,

Some of our instance's users are not familiar with mCPU, would it be possible to explain what it means on a kubernetes-based node on the service "resources" tab on the launching view? Or point to external resource with a "details" icon?

For the same kind of reason, it is possible to specify the memory is RAM memory in this tab?

**Comment by fcomte** [comment]
I think this is more related to the helm chart. 

https://github.com/InseeFrLab/helm-charts-interactive-services/blob/89ee338009f0fd8cf3d55de0f697265f2a283544/charts/jupyter-python/values.schema.json#L113

I think we can provide markdown link in the description ? @garronej do you confirm ?

**Comment by garronej** [comment]
Not yet as I just introduced the <LocalizedMarkdown /> component but it can be in a matter of seconds.

**Comment by garronej** [comment]
Just need to replace `<Text>{field.description}</Text>` by `<LocalizedMarkdown>{field.description}</LocalizedMarkdown>`

**Comment by qgau** [comment]
Hello, to give you the bottom line on this topic. In the end we decided to keep mcpu in the service configuration because 1. we could benefit of other charts (like SSPCloud's), 2. we think it is better to stay harmonized with the community, and 3. we manage to understand the root issue for our users (mostly scientists) was to understand if the services they use (or implement) could benefit from multi-threading (virtual or real, etc.).
So in the end we made a dedicated documentation for that, recalling the big principles of Kubernetes clusters and mCPU, as well as describing our underlying nodes so that they have a better understanding of what is happening under the hood.

I am closing the issue then, thank you for the tips and sorry for the late feedback.

PS: we thought about addding automatic node descriptions somewhere in the onyxia interface (in the quotas part), but we though it was not really relevant, because 1. only a small set of users asked us this info, and 2. with particular kubernetes configuration (in our cases, for example, multiple node flavors, gpus, etc.) it can be difficult to simply show node descriptions without other comments. In addition, if some users of us ask to deploy distributed computing clusters (such as Dask or spark), it will be even more complicated to explain.
